src/hooks/balances/resolvers/query/balances.test.tsx

```diff
@@ -0,0 +1,272 @@
+import {
+  balancesByAddressQueryResolverFactory,
+  BalancesByAddressResolverArgs,
+  Entity,
+  objectToArrayWithoutNull,
+  useBalanceQueryResolvers,
+} from './balances';
+
+import { AssetIds, Balance } from '../../../../generated/graphql';
+import { Resolvers, useQuery } from '@apollo/client';
+import { MockedProvider } from '@apollo/client/testing';
+import TestRenderer, { act } from 'react-test-renderer';
+import { gql } from 'graphql.macro';
+import {
+  getMockApiPromise,
+  nativeAssetBalance,
+  nonNativeAssetBalance,
+} from '../../../polkadotJs/tests/mockUsePolkadotJsContext';
+import { ApiPromise } from '@polkadot/api';
+import waitForExpect from 'wait-for-expect';
+import errors from '../../../../errors';
+waitForExpect.defaults.interval = 1;
```

Comment:
i'd move this to a more global-ish config, so that all tests will benefit from it.

But considering how jest loads tests, it is probably fine here for now.

Reply:
I have moved it to global and it is imported through the setupFiles config step